docs/_includes/attr-data.adoc

```diff
@@ -2,58 +2,141 @@
 Included in:
 
 - user-manual: Built-in data attributes
+
+Changes:
+Added an example/notes column; 
+deleted doctitle because it is documented elsewhere; 
+added basebackend, filetype and outdir;
+consistently use 'document' for the input file and 'output file' for the output.
+added safe-mode stuff
 ////
 
-Asciidoctor includes numerous intrinsic attributes that are assigned values when the document is rendered.
-The values of these built-in data attributes are derived from how the document is processed and when and where is it processed.
+Asciidoctor includes numerous attributes that are assigned values when the document is rendered.
+The values of these attributes are derived from how the document is processed and when and where is it processed.
 These attributes can be referenced anywhere in the document.
 
 // tag::table[]
-.Built-in data attributes
-[width="70%",cols="1m,3"]
+.Document attributes - runtime information
+[cols="1m,3a,1m"]
 |===
-|Attribute |Description
+|Attribute |Description |Example
 
-|asciidoctor
-|Calls the processor
+|asciidoctor 
+|Defined if the current processor is Asciidoctor.
+//|{asciidoctor}
+|
 
-|asciidoctor-version
-|Version of the processor
+|asciidoctor-version 
+|Version of the processor.
+//|Example: {asciidoctor-version}
+|1.5.4
 
 |backend
-|Backend used to render document
+|Backend used to create the output file.
+//|Example: {backend}
+|html5
+
+|basebackend
+|`html` or `docbook`.
```

Comment:
basebackend is the backend value minus any trailing numbers (html5 -> html5, docbook5 -> docbook).